channels/channel.nos/nos2010/chn_nos2010.py

```diff
@@ -1165,45 +1165,44 @@ def update_video_item_live(self, item: MediaItem) -> MediaItem:
         return item
 
     def create_iptv_streams(self, parameter_parser):
-        """ Fetch the available live channels using EPG endpoint and format them into JSON-STREAMS
+        """ Fetch the available live channels using guide-channels endpoint and format them into JSON-STREAMS
 
         :param ActionParser parameter_parser: a ActionParser object to is used to parse and
                                                    create urls
 
         :return: Formatted stations
         :rtype: list
         """
-        epg_url = datetime.datetime.now().strftime("https://start-api.npo.nl/epg/%Y-%m-%d?type=tv")
-        epg_data = UriHandler.open(epg_url,
-                                   no_cache=True,
-                                   additional_headers=self.__jsonApiKeyHeader)
-        epg = JsonHelper(epg_data)
-
+        channel_data = JsonHelper(UriHandler.open(f"https://npo.nl/start/api/domain/guide-channels"))
         parent_item = MediaItem("Live", "https://www.npostart.nl/live", media_type=mediatype.FOLDER)
         items = []
         iptv_streams = []
 
-        for stations in epg.get_value("epg"):
-            livestream = JsonHelper.get_from(stations, "channel", "liveStream")
-            item = MediaItem(JsonHelper.get_from(livestream, "title"),
-                             JsonHelper.get_from(livestream, "shareUrl"),
-                             media_type=mediatype.VIDEO)
-            item.isLive = True
-            item.isGeoLocked = True
+        # NPO start switched to SVG logos embedded in code. So, get them from Wikipedia instead, which should be stable-ish.
+        # 800px width recommended https://kodi.wiki/view/Live_TV_Artwork
+        logo_sources = {
+            "NPO1": "https://upload.wikimedia.org/wikipedia/commons/thumb/0/02/NPO_1_logo_2014.svg/800px-NPO_1_logo_2014.svg.png",
```

Comment:
Please use the built-in images: TextureHandler.instance().get_texture_uri(self, "<image name>")

Reply:
Sure!